ONSdigital/dp-filter-api
# Feature/pagination update

### What

Updated paginated endpoints to comply with the latest API standards defined in https://github.com/ONSdigital/dp/blob/master/standards/API_STANDARDS.md :
- limit=0 returns an empty list of items instead of all items
- default limit = 20
- cap maximum limit to 1000
- negative values for offset and limit result in a 400 Bad Request response
- Updated swagger spec

### How to review

- Make sure changes comply with the new standards
- Make sure changes make sense
- Make sure unit tests pass

### Who can review

Anyone
